### DISCUSSION TITLE
Cross compile with CGO

### DISCUSSION THREAD

**Discussion by jmoney** [goreleaser/goreleaser#4903] [Ideas, Issue Triage, and General discussions]
Trying to cross compile in a github action to create binary that's completely self contained.  I followed the instructions https://github.com/plentico/osxcross-target?tab=readme-ov-file and got things mostly working but there's one error i do not know what it means: https://github.com/jmoney/tokenizer-utils/actions/runs/9327636534/job/25677789866.  

`/usr/bin/ld: Error: unable to disambiguate: -arch (did you mean --arch ?)`.  I'm unsure what variables that need to be updated

**Comment by caarlos0** [comment]
the actions run link is a 404, and I dunno about osxcross-target.

the error seems to be some difference between flags in bsd ld and gnu ld? no idea how to fix it though.

FWIW its probably easier to follow https://goreleaser.com/cookbooks/cgo-and-crosscompiling/

**Comment by jmoney** [comment]
Perfect! Thanks. That worked perfectly.
